ups/ts_cbp.table

```diff
@@ -5,13 +5,10 @@
 # included by low-level LSST packages such as utils or daf_base.
 # Any other package whose interface is used should be listed explicitly
 # rather than assuming it will be included recursively.
-setupRequired(boost >= 1.47.0)
-setupRequired(utils >= 4.6.0.0)
-setupRequired(pex_exceptions >= 4.6.0.0)
 
 
 # The following is boilerplate for all packages.
 # See Tech Note DMTN-001 for details on LSST_LIBRARY_PATH
 # Pure-Python packages only need the last line.
-envPrepend(PYTHONPATH, ${PRODUCT_DIR}/lsst)
+envPrepend(PYTHONPATH, ${PRODUCT_DIR}/python)
```

Comment:
Instead of "scripts" on the next line please consider "bin" for command-line scripts, like all our other packages.